lang/en.json

```diff
@@ -13,16 +12,17 @@
 	"ITEM.TypeAttack": "Attack",
 	"ITEM.TypeCypher": "Cypher",
 	"ITEM.TypeEquipment": "Equipment",
-	"ITEM.TypeLasting Damage": "Lasting Damage",
+	"ITEM.TypeLastingDamage": "Lasting Damage",
 	"ITEM.TypeMaterial": "Crafting Material",
 	"ITEM.TypeOddity": "Oddity",
-	"ITEM.TypePower Shift": "Power Shift",
+	"ITEM.TypePowerShift": "Power Shift",
 	"ITEM.TypeSkill": "Skill",
-	"ITEM.TypeTeen Ability": "Ability (Teen)",
-	"ITEM.TypeTeen Armor": "Armor (Teen)",
-	"ITEM.TypeTeen Attack": "Attack (Teen)",
-	"ITEM.TypeTeen Lasting Damage": "Lasting Damage (Teen)",
-	"ITEM.TypeTeen Skill": "Skill (Teen)",
+	"ITEM.TypeTeenAbility": "Ability (Teen)",
+	"ITEM.TypeTeenArmor": "Armor (Teen)",
+	"ITEM.TypeTeenAttack": "Attack (Teen)",
+	"ITEM.TypeTeenLastingDamage": "Lasting Damage (Teen)",
+	"ITEM.TypeTeenSkill": "Skill (Teen)",
+	"ITEM.NewItem": "[New {item}]",
```

Comment:
That’s on purpose. There really is a space in the type names.

Reply:
Yeah but it will not work for the localisation thing (at least not when I tried it) if you do ` game.i18n.localize("ITEM.TypeTeen Skill")` :)

Reply:
Also, we can’t use "ITEM.NewItem": "[New {item}]". Some languages, German for example, has gendered nouns, so we need a separate string including “New” for all Actors and Items.

Example: It’s “Neue**r** Gegenstand,” “Neu**e** Fertigkeit,” and “Neue**s** Artefakt” in German.